signac/common/config.py

```diff
@@ -78,10 +90,11 @@ def read_config_file(filename, configspec=None, *args, **kwargs):
     try:
         config = Config(filename, configspec=configspec, *args, **kwargs)
     except (OSError, ConfigObjError) as error:
-        msg = "Failed to read configuration file '{}':\n{}"
-        raise ConfigError(msg.format(filename, error))
+        raise ConfigError(f"Failed to read configuration file '{filename}':\n{error}")
     verification = config.verify()
     if verification is not True:
+        # TODO: In the future this should raise an error, not just a
```

Comment:
Is this a "2.0 requirement" future or some arbitrary future? Is there a problem with raising now?

Reply:
This is a "out of scope for this PR" TODO. I don't plan to address this until switching to TOML, no point doing it with the configobj API since TOML is likely to look somewhat different anyway.